base/linalg/lapack.jl

```diff
@@ -5299,7 +5299,7 @@ for (bdsdc, elty) in
                 u, &ldu, vt, &ldvt,
                 q, iq, work, iwork, info)
             chklapackerror(info[])
-            d, e, u, vt, q, iq
+            d, e_, u, vt, q, iq
```

Comment:
Apparently we returned that mathematical constant `e` from `LAPACK.bdsdc!`. :metal:

Reply:
Just checked and `e_` is just garbage after the call to `xbdsdc` so maybe just delete it (and adjust `svd`)